charts/mirrorbits-lite/Chart.yaml

```diff
@@ -1,4 +1,7 @@
-apiVersion: v1
+appVersion: v0.5.2
```

Comment:
```suggestion
apiVersion: v1
appVersion: v0.5.2
```

Reply:
```suggestion
```
And appVersion isn't (nit)

Reply:
Good catch for the confusion I had between `appVersion` and `apiVersion` Thanks!

`appVersion` indicate the version of the application provided, which is (and should) always be different than the version of the Docker image. In our case it is definitively different as the version schem of jenkinsciinfra/mirrorbits is different than mirrorbits version scheme

Reply:
Fixed by https://github.com/jenkins-infra/helm-charts/pull/731/commits/d9f4998bc4b6485670cf494d39d4c4cc4b5c793b